server/protocols/webadmin/webadmin-data/src/main/java/org/apache/james/webadmin/routes/DomainsRoutes.java

```diff
@@ -293,16 +302,15 @@ private Response exists(Request request, Response response) throws DomainListExc
         }
     }
 
-    private ImmutableSet<DomainAliasResponse> listDomainAliases(Request request, Response response) throws DomainListException, RecipientRewriteTableException {
+    private ImmutableList<DomainAliasResponse> listDomainAliases(Request request, Response response) throws DomainListException, RecipientRewriteTableException {
```

Comment:
Why have you dropped the uniqueness property?

Reply:
Well technically I did not drop the uniqueness property - it was still tested.

But it was less explicit.

To make it more clear I turned DomainAliasResponse into a POJO and turned that back into a Set